plugins/woocommerce-admin/client/products/product-variation-form.tsx

```diff
@@ -50,7 +50,7 @@ export const ProductVariationForm: React.FC< {
 			ref={ formRef }
 		>
 			<ProductVariationFormHeader />
-			<ProductFormLayout>
+			<ProductFormLayout key={ productVariation.id }>
```

Comment:
I think this might be the key to some of our update issues instead of calling `resetForm`.